src/v/cluster/topic_configuration.h

```diff
@@ -67,11 +69,14 @@ struct topic_configuration
     int32_t partition_count;
     // using signed integer because Kafka protocol defines it as signed int
     int16_t replication_factor;
+    // bypass migration restrictions
+    bool is_migrated;
```

Comment:
i am wondering if we should make this property a part of `topic_properties` and allow it to be described from the Kafka APIs

Reply:
> i am wondering if we should make this property a part of topic_properties

I'm not quite sure. What's the difference between the two?
> and allow it to be described from the Kafka APIs

Why would we need it?

TBH I'd hide it as much as possible, as really it's only needed as a flag to create it. It does not make much sense afterwards, as a migrated topic should not be different from one created the normal way.